scaleway/resource_iot_device.go

```diff
@@ -141,18 +130,31 @@ func resourceScalewayIotDevice() *schema.Resource {
 					Schema: map[string]*schema.Schema{
 						"crt": {
 							Type:        schema.TypeString,
+							Optional:    true,
 							Computed:    true,
```

Comment:
```suggestion
							Optional:    true,
							ForceNew:      true,
							Computed:    true,
```
 Please consider add ForceNew

Reply:
As discussed we do not want to trigger the certificate re-creation if the user updates its certificate.